speed-dreams/source-2.2.3/src/libs/tgfclient/tgfclient.h

```diff
@@ -476,6 +479,9 @@ TGFCLIENT_API void GfuiUnSelectCurrent(void);
 
 TGFCLIENT_API int  GfuiFontHeight(int font);
 TGFCLIENT_API int  GfuiFontWidth(int font, const char* text);
+
+// SIMULATED DRIVING ASSISTANCE: expose method to API.
+TGFCLIENT_API void GfuiDrawString(int p_x, int p_y, const char* p_string, GfuiFontClass* p_font, float* p_color);
 TGFCLIENT_API void GfuiDrawString(const char* text, float* fgColor, int font,
                                   int x, int y, int width = 0, int hAlign = GFUI_ALIGN_HL);
```

Comment:
add new GfuiDrawString function to CHANGES.txt

Reply:
Already in there (see last commit before the PR opened for review)